docs/pages/es/architecture.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Arquitectura de Meshery 
+permalink: es/architecture
+---
+
+## Arquitectura
+
+#### La arquitectura de Meshery puede ser vista desde dos perspectivas:
```

Comment:
Thank you for this, @Drake8710. 

Let's avoid constraining the architectural perspective of Meshery to only two perspectives, understanding that there are a number of diagrams that we will publish over time.